core/model/__init__.py

```diff
@@ -211,13 +211,13 @@ class PresentationCalculationPolicy:
 
     def __init__(
         self,
+        /,
```

Comment:
@dbernstein it was intentional here, but I actually messed it up :facepalm:, so its a good thing you asked about it!  

I meant to mark these as keyword only parameters, since that is how they have been used anyway. I got it backwards though, `/` is the mark for positional only parameters, and `*` is for keyword only. So this effectively did nothing. I pushed a commit changing it to `*` to mark these as keyword only. 

Python docs have some more info:
https://docs.python.org/3/tutorial/controlflow.html#special-parameters

Reply:
Thanks for the python tip.